app/views/merchants/items/show.html.erb

```diff
@@ -1,4 +1,6 @@
 <h2>Item Attributes</h2>
-<p> Name:<%= @item.name %></p>
-<p>Description:<%= @item.description %></p>
-<p>Current Selling Price:<%= @item.unit_price %></p>
\ No newline at end of file
+
+  <p> Name:<%= @item.name %></p>
+  <p>Description:<%= @item.description %></p>
+  <p>Current Selling Price:<%= @item.unit_price %></p>
+  <p><%= link_to "Update #{@item.name}", edit_merchant_item_path(@merchant, @item) %></p>
```

Comment:
the wireframe shows the button can just be "Update Item" for every item (dont need to interpolate in the item's name on the button)